includes/utils/class-amp-carousel.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Class AMP_Carousel.
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Carousel
+ *
+ * Gets the markup for an <amp-carousel>.
+ *
+ * @since 1.4.1
```

Comment:
Maybe this will be different.

Reply:
Yes, let's have the changes here by targeting 1.5.0

Reply:
Sounds good